compute_endpoint/globus_compute_endpoint/endpoint/interchange.py

```diff
@@ -388,9 +389,15 @@ def process_pending_tasks() -> None:
 
                     fid: str = prop_headers.get("function_uuid")
                     tid: str = prop_headers.get("task_uuid")
+
                     if not fid or not tid:
                         raise InvalidMessageError("Task message missing headers")
```

Comment:
It looks like this exception would be logged and the interchange would continue, which I think is bad handling. If the web-service sent out malformed headers, the interchange should send out a response, and shutdown.

Reply:
Hmm; let's talk, because that was very much the original intention.  To shut down instead assumes that there's a fundamental web-service error and/or that this scenario would be common.  In my head, I expect this to ~never occur, or to be a fluke (perhaps bit-rot?) every now and then.  I think the better part of valor is to log the invalid packet and continue processing, no?

Reply:
Update message to say missing tid or fid.